go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	code.cloudfoundry.org/rep v0.0.0-20200325195957-1404b978e31e // indirect
 	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
 	code.cloudfoundry.org/tlsconfig v0.0.0-20200131000646-bbe0f8da39b3 // indirect
-	github.com/DataDog/agent-payload v4.43.0+incompatible
+	github.com/DataDog/agent-payload v4.43.1-0.20200901084552-bf35d9cd9b96+incompatible
```

Comment:
will be updated as soon as https://github.com/DataDog/agent-payload/pull/67 gets merged